src/main/java/core/basesyntax/handler/ConsoleHandler.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.handler;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler implements Handler {
+    private BetDao betDao = new BetDaoImpl();
+
+    @Override
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+
+        while (true) {
+            Bet bet = null;
+            String input = scanner.nextLine();
+            if (input.equalsIgnoreCase("q")) {
+                return;
+            }
+            try {
+                String[] parseString = input.split(" ");
+                int value = Integer.parseInt(parseString[0]);
+                double risk = Integer.parseInt(parseString[1]);
+
+                if (risk < 0 || value < 0) {
+                    System.out.println("Value is negative");
```

Comment:
should we skip the rest of code in the current iteration?

Reply:
Above you wrote to me that i continue my implementation. I'm a little confused already :)

Reply:
Okey i throw RunTimeException